packages/muelu/src/Graph/Containers/MueLu_LWGraph_kokkos_def.hpp

```diff
@@ -99,6 +99,32 @@ namespace MueLu {
     Kokkos::parallel_reduce("MueLu:LWGraph:LWGraph:maxnonzeros", range_type(0,graph_.numRows()), maxNumRowEntriesFunctor, maxNumRowEntries_);
   }
 
-}
+  template<class LocalOrdinal, class GlobalOrdinal, class DeviceType>
+  void LWGraph_kokkos<LocalOrdinal,GlobalOrdinal,Kokkos::Compat::KokkosDeviceWrapperNode<DeviceType>>::
+  print(Teuchos::FancyOStream &out, const VerbLevel verbLevel) const {
+
+    if (verbLevel & Debug) {
+      RCP<const Map> col_map = importMap_.is_null() ? domainMap_ : importMap_;
+      int mypid = col_map->getComm()->getRank();
+
+      out << "[pid " << mypid << "] num entries=" << graph_.entries.size() << std::endl;
+
+      const size_t numRows = graph_.numRows();
+      auto rowPtrs = graph_.row_map;
+      auto columns = graph_.entries;
+      for (size_t i=0; i < numRows; ++i) {
```

Comment:
To help a little with mangled outputs I like to use `std::ostringstream` and then output the whole thing at once.
But that's implementation is fine too

Reply:
That's a good point.

Reply:
fixed